setup.cfg

```diff
@@ -30,3 +30,12 @@ console_scripts =
 
 [wheel]
 universal = 1
+
+[flake8]
+builtins = _
```

Comment:
is this doing anything? I didn't see any usage of global `_` at a glance

Reply:
This is the OpenStack default for extending builtins for doing internationalization (i18n)